src/components/VoiceRecorder/VoiceRecorder.reducer.js

```diff
@@ -0,0 +1,18 @@
+const initialState = { audioURL: '', iconsColor: 'black' };
+
+function OnRecordClick(state = initialState, action) {
+  switch (action.type) {
+    case 'ADD_RECORD':
```

Comment:
Should import those case strings like `ADD_RECORD` from the `.constants` file.
The benefit in using strings from a single source of truth is when we refactor and rename our strings, we don't need to do it on each component using them.